### ISSUE TITLE
Create functionality and UI for "Login" view

### ISSUE THREAD

**Issue by MiguelTepale** [MiguelTepale/iOS-Studies#9]
Instructions are available in LoginViewController.swift

**Comment by MiguelTepale** [comment]
@nitisha-neuroleap, please make sure to pull the recent changes before starting any work on a new branch.

**Comment by nitisha-neuroleap** [comment]
I’m working on the buttons with auto layout right now, I’m almost done with
that. I set up a branch for that already. That one will not be merged into
master, right?

**Comment by MiguelTepale** [comment]
> I’m working on the buttons with auto layout right now, I’m almost done with that. I set up a branch for that already. That one will not be merged into master, right?
> […](#)

What buttons are you referring to? Please indicate the view controller.

**Comment by nitisha-neuroleap** [comment]
The ones on the MenuViewController, that you said I should try making using
only auto layout.

**Comment by MiguelTepale** [comment]
Amazing! Let's continue the conversation of MenuViewController.swift in #10. The current thread is to discuss the "Login" view.
